components/segmenter/src/word.rs

```diff
@@ -435,8 +486,14 @@ impl WordSegmenter {
             data: self.payload.get(),
             complex: Some(&self.complex),
             boundary_property: 0,
+            locale_override,
         })
     }
+
+    fn is_default_rule(locale: &DataLocale) -> bool {
+        let lang = locale.language();
+        lang != language!("fi") && lang != language!("sv")
```

Comment:
Same here: please just try to load from data.

It would be okay to skip `und`, but please note that in the issue we agreed that the function should take an _optional_ locale, and you should skip loading when the locale is None.

Reply:
Nit: Please just always check the data provider. Don't rely on a hard-coded list of locales.